.github/ISSUE_TEMPLATE/enhancement-new-base-rule-extension.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Enhancement: Propose a New Base Rule Extension"
+description: "Propose a new base lint rule extension"
+title: "[rule-name] a short description of my proposal"
+labels:
+  - "enhancement: new base rule extension"
+  - "package: eslint-plugin"
+  - triage
+body:
+  - type: markdown
+    id: preamble
+    attributes:
+      value: |
+        # What is a "Base Rule Extension"?
+        In some cases, ESLint provides a rule itself, but it doesn't support TypeScript syntax;
+        either it crashes, or it ignores the syntax, or it falsely reports against it.
+        In these cases, we create what we call an extension rule; a rule within our plugin that
+        has the same functionality, but also supports TypeScript.
```

Comment:
This paragraph is rendering a little weird because of the newline breaks:

![Screenshot of the four lines each being a paragraph in the rendered output](https://user-images.githubusercontent.com/3335181/166967225-cced33be-f8a5-48c1-b555-9206c5f2649e.png)

Does something like `\` at the end of the line work to make it a single paragraph?